akaunting/akaunting_stackscript.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# <UDF name="company_name" Label="Company Name" default="My Company" />
+# <UDF name="company_email" Label="Company Email" default="my@company.com" />
+# <UDF name="admin_email" Label="Admin Email" default="my@company.com" />
```

Comment:
From our experience, users will typically enter the same email when asking for multiple emails. We'd recommend only including 1 email UDF and using that email for both company and admin email.